modules/support-submitting-a-case.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * support/getting-support.adoc
+
+[id="support-submitting-a-case_{context}"]
+= Submitting a support case
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+* You have installed the OpenShift CLI (`oc`).
+* You have a Red Hat Customer Portal account.
+* You have a Red Hat standard or premium subscription.
+
+.Procedure
+
+. Login to the link:http://access.redhat.com[Red Hat Customer Portal] and select *SUPPORT CASES* -> *Open a case*.
```

Comment:
s/Login/log in/
**log in** _v_
Write as shown when used as a verb - two words.

Reply:
Thanks Bob. Resolved.